### ISSUE TITLE
Error: Cannot read property 'properties' of undefined

### ISSUE THREAD

**Issue by mikehuebner** [venits/react-native-map-clustering#163]
Hello!

Just started using this for a project and I noticed that the spiral logic doesn't seem to be checking for undefined items. Why these items are undefined I am not sure but they are. Here is the full error plus how to fix if anyone needs it.

```
TypeError: Cannot read property 'properties' of undefined
  at generateSpiral (VM16 helpers.bundle:76)
  at ClusteredMapView.js:117
```

Within helpers.js (generateSpiral) I just added this
```
for (let i = 0; i < count; i++) {
    angle = 0.25 * (i * 0.5);
    let latitude = centerLocation[1] + 0.0002 * angle * Math.cos(angle);
    let longitude = centerLocation[0] + 0.0002 * angle * Math.sin(angle);

    if (clusterChildren[i + start]) {
      res.push({
        index: clusterChildren[i + start].properties.index,
        longitude,
        latitude,
        centerPoint: {
          latitude: centerLocation[1],
          longitude: centerLocation[0],
        },
      });
    }
  }
```

and that seemed to fix it, but unsure if that is a real solution.

**Comment by venits** [comment]
Hi @mikehuebner 

hmm that's strange, some weird combinations of markers must rigger that 🤔

I released version `3.3.7` with your fix 🎉

**Comment by markrickert** [comment]
Holy hell, you're fast @venits!
